addons/api/addon/generated/models/account.js

```diff
@@ -43,4 +43,20 @@ export default class GeneratedAccountModel extends BaseModel {
     description: 'Current version number of this resource.',
   })
   version;
+
+  // =attributes (password)
+  @attr('string', {
+    for: 'password',
+    isNestedAttribute: true,
+    description: 'The account login name',
+  })
+  login_name;
+
+  // =attributes (OIDC)
+  @attr('string', { for: 'oidc', isNestedAttribute: true }) subject;
```

Comment:
In other fields on this file, or in other files on this PR I see when defining `@attr` we use several lines rather than one liner.

I do not have a strong opinion here, but wondering if we should be consistent with it?

Reply:
This is automated via prettier.  If the line exceeds the max line length, it gets split.